QgisModelBaker/libili2db/globals.py

```diff
@@ -17,8 +17,30 @@
  *                                                                         *
  ***************************************************************************/
 """
+from enum import IntFlag
 
 CRS_PATTERNS = {
     'LV95': 2056,
     'LV03': 21781
 }
+
+class DbIliMode(IntFlag):
+    pg = 1
+    gpkg = 2
+    mssql = 4
+
+    ili = 1024
+
+    ili2pg = ili | pg
+    ili2gpkg = ili | gpkg
+    ili2mssql = ili | mssql
+
+displayDbIliMode = {
+    DbIliMode.pg: "PostgreSQL/PostGIS",
```

Comment:
translation should happen here. 
```suggestion
    DbIliMode.pg: QCoreApplication.translate("PostgreSQL/PostGIS"),
```